gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -65,7 +65,6 @@
 _VARIANT_MERGE_STRATEGIES = [
     _NONE_STRING,
     _MOVE_TO_CALLS_STRING,
-    _MERGE_WITH_NON_VARIANTS_STRING,
```

Comment:
nit: please add a short comment that this is under development in the comment above.

Reply:
Is it okay to just delete the part about merge_with_non_variants and then add it back once things are finished?

Reply:
sg :)